src/kaltura-player.js

```diff
@@ -656,6 +687,19 @@ class KalturaPlayer extends FakeEventTarget {
     }
   }
 
+  _onChangeSourceEnded(): void {
+    if (Utils.Object.getPropertyPath(this.config, 'ui.targetId')) {
```

Comment:
Add warning to console if not initialising

Reply:
done